stdlib/2and3/codecs.pyi

```diff
@@ -31,6 +31,7 @@ class _IncrementalDecoder(Protocol):
 def encode(obj: _Decoded, encoding: str = ..., errors: str = ...) -> _Encoded: ...
 def decode(obj: _Encoded, encoding: str = ..., errors: str = ...) -> _Decoded: ...
 def lookup(encoding: str) -> CodecInfo: ...
+def utf_16_be_encode(obj: _Decoded, errors: str = ...) -> _Encoded: ...  # undocumented
```

Comment:
While you're at it should we also add the corresponding decode method?

Also, both of the arguments are positional-only, so they should be prefixed with __.

Reply:
Done and done.